bin/zeppelin.sh

```diff
@@ -68,6 +68,9 @@ do
     -h)
         usage
         ;;
+    *)
+        usage
+        ;;
```

Comment:
It could be an unintended mistake by the user, so it would be better if the script exited with a non-zero value.

Reply:
Agreed.

What do you think about 

```bash
        echo "Unsupported argument. Please, call with -h/--help for usage information"
        exit 1
        ;;
```

Reply:
It would be nice if the following items could be modified as well. What do you think?
- Combine `--help` and `-h` processing into one
- Separate the `exit` call part from the `usage` function.
- `*)` In the processing, call the `usage` function after `echo "Unsupported argument."`

Reply:
Done!